BREEZE.rst

```diff
@@ -474,61 +453,47 @@ Run pylint checks for all files:
      ./breeze --static-check-all-files pylint
 
 
-The ``license`` check is also run via separate script and separate docker image containing
+The ``license`` check is run via a separate script and a separate docker image containing the 
 Apache RAT verification tool that checks for Apache-compatibility of licences within the codebase.
-It does not take pre-commit parameters as extra args.
+It does not take pre-commit parameters as extra arguments.
 
 .. code-block:: bash
 
      ./breeze --static-check-all-files licenses
 
-Building the documentation
---------------------------
-
-The documentation is build using ``-O``, ``--build-docs`` command:
-
-.. code-block:: bash
-
-     ./breeze --build-docs
-
-Results of the build can be found in ``docs/_build`` folder. Often errors during documentation generation
-come from the docstrings of auto-api generated classes. During the docs building auto-api generated
-files are stored in ``docs/_api`` folder - so that in case of problems with documentation you can
-find where the problems with documentation originated from.
-
-Running tests directly from host
---------------------------------
+Running Tests Directly from the Host
```

Comment:
I don't see any 'from the host' specifics below. does it mean running tests outside the container?

Reply:
It's bad naming indeed. The tests are still running in the container, but they are triggered from the host. You do not need to enter container separately for that - the script will start and enter container automatically.